misc/scripts/install-local.sh

```diff
@@ -216,6 +251,8 @@ if [[ $NOBUILDDEP -eq 0 ]]; then
 	fi
 fi
 
+makeVirtualDeb
```

Comment:
Add fancy messages for creating dummy package and for installing dependencies

Reply:
Also, add the dummy deb removal in `trap_ctrl_c` if the dummy has already been installed. A simple variable check should do the job

Reply:
And add the dummy deb removal before every `return 1` that comes after the `makeVirtualDeb` call